lib/web_ui/lib/src/engine/scene_view.dart

```diff
@@ -301,13 +345,61 @@ final class PlatformViewContainer extends SliceContainer {
     }
   }
 
+  set clipPath(ScenePath? path) {
+    if (_clipPath == path) {
+      return;
+    }
+
+    if (_clipPath != null) {
+      _clipRegistry.unregisterPath(_clipPathId!);
+      _clipPathId = null;
+    }
+
+    if (path != null) {
+      _clipPath = path;
+      _clipPathId = _clipRegistry.registerPath(path);
+    }
+  }
+
+  String cssStringForClip(PlatformViewClip clip, double devicePixelRatio) {
+    switch (clip) {
+      case PlatformViewNoClip():
+        clipPath = null;
+        return '';
```

Comment:
The function could instead return `null`, and the place that applies the clip could avoid setting the property to an empty string unnecessarily.

Reply:
The container can be reused when the styling changes, so we actually *do* need to set the property to an empty string if the clipping goes away.